lib/elixir/lib/exception.ex

```diff
@@ -1648,12 +1650,25 @@ defmodule UndefinedFunctionError do
   end
 
   defp hint(module, function, arity, :suggest_function) do
-    behaviour_hint(module, function, arity) <>
+    hint_for_behaviour(module, function, arity) <>
       hint_for_loaded_module(module, function, arity, nil)
   end
 
-  @max_suggestions 5
   defp hint(module, function, arity, :suggest_module) do
+    hint_for_missing_module(module, function, arity)
+  end
+
+  defp hint_for_missing_alias(module) do
+    with "Elixir." <> rest <- Atom.to_string(module),
+         false <- rest =~ "." do
+      ". Make sure the module name is correct and that it has the proper namespace (if any)"
```

Comment:
I would appreciate feedback on this message.

Reply:
@josevalim I’m not too clear on what "has the proper namespace" means here. I think maybe we wanna say something like "make sure it's correct and it's been properly aliased" or the likes. Sounds horrible but maybe mentioning "alias" somehow would help clarify?

Reply:
But the issue is that we don't want people to think they _have_ to define an alias. What if we say:

    Make sure the module name is correct and has been specified in full (or that an alias has been defined)

Reply:
Yes perfect!